docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,53 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
+
+Upgrading Alluxio in Kubernetes is easy. You can just change the Alluxio docker image version to your desired upper version.
+Then restart the Pod to make it take effect. You should make sure the Alluxio masters and workers are running on the same version for the best compatibility.
+
+For example, if you are currently running Alluxio v2.0.1 on Alluxio masters and you want to upgrade to the latest version.
+Just change the `image` field of all the Alluxio containers to `image: alluxio/alluxio:latest` and restart them.
+
+```yaml
+  containers:
+  - name: alluxio-master
+    image: alluxio/alluxio:2.0.1
```

Comment:
the tag is not `latest` as per the line above

Reply:
I was trying to say "Find the `image` lines in the YAML below and change them to `latest`". But since it is more intuitive that the code snippet shows the desired resulting YAML, I'll update that.